src/python/pants/help/build_dictionary_info_extracter.py

```diff
@@ -175,7 +177,7 @@ def get_function_args(cls, func):
   @classmethod
   def _get_function_args(cls, func):
     arg_descriptions = cls.get_arg_descriptions_from_docstring(func)
-    argspec = inspect.getargspec(func)
+    argspec = inspect.getfullargspec(func) if PY3 else inspect.getargspec(func)
```

Comment:
Was this causing a failure or causing us to output `DeprecationWarning`?

Reply:
It was outputting a `DeprecationWarning`, and we had a test asserting our logs had no warnings. I thought it would be better to just replace it with the recommended new function. Hopefully the use of `PY3` will make it easy to grep and clean up when we remove py2k support.

Reply:
Gotcha. We've had a couple of failures like that in other places.

Agreed this is definitely the best solution. Silencing the error would have led to quite the fun debugging episode when Python 3.8 comes out and they remove the original function 😶

Thanks!